packages/fxa-shared/subscriptions/configuration/helpers.ts

```diff
@@ -0,0 +1,164 @@
+import parser from 'accept-language-parser';
+
+import {
+  DEFAULT_PRODUCT_DETAILS,
+  metadataFromPlan,
+  productDetailsFromPlan,
+} from '../metadata';
+import { Plan } from '../types';
+import { ProductConfig } from './product';
+
+const DEFAULT_LOCALE = 'en';
+
+export const getPlanProductConfig = (plan: Plan) => {
+  const configuration = plan.configuration;
+
+  if (!configuration) {
+    throw new Error(`Plan configuration not found. (${plan.plan_id})`);
```

Comment:
nit: I think usually we include the id in the message to provide some context.  e.g. "plan XYZ not found" or "config for plan ABC not found".  not an issue in this case since the format of the id is either "price_*" or "plan_*" so we know what sort of id it is.

Reply:
_/me googles "how to include italics in JS error message"_ 😅